.gitignore

```diff
@@ -1,12 +1,14 @@
-*.class
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.ear
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
```

Comment:
Because I was tired and just copied the .gitignore from wicketstuff-core without thinking. Re-adding those lines now.